src/app/containers/ConsentBanner/index.amp.jsx

```diff
@@ -21,7 +22,7 @@ const jsonInlinedScript = data => (
   <script
     type="application/json"
     /* eslint-disable-next-line react/no-danger */
-    dangerouslySetInnerHTML={{ __html: JSON.stringify(data) }}
+    dangerouslySetInnerHTML={{ __html: xssJsonStringify(data) }}
```

Comment:
I'm not sure you need to use `dangerouslySetInnerHTML` here since. I think you can just do:

```jsx
const jsonInlinedScript = data => (
  <script type="application/json">{JSON.stringify(data)}</script>
);
```